tests/bounties/solidity-bounty/start.sh

```diff
@@ -1,4 +1,44 @@
 #!/bin/sh
+
+#
+# Validate exploit code - Exploit code file must contain only ascii characters
+#
+grep -q "[^[:print:]]" $1 
+status=$?
+if [ $status -eq 0 ]; then
+    >&2 echo "Invalid exploit code: File contains non-ascii characters"
+    exit 1
+fi
+if [ $status -eq 2 ]; then
+    >&2 echo "Invalid exploit code: Error reading the file"
+    exit 1
+fi
+
+
+#
+# Validate exploit code - No experimental directives are allowed
+#
+grep -q "experimental" $1
+status=$?
+if [ $status -eq 0 ]; then
+    >&2 echo "Invalid exploit code: contains 'experimental' keyword"
+    exit 1
+fi
+
+
+#
+# Validate exploit code - Requires specific version of Solidity
+#
+grep -q "pragma solidity 0.8.26" $1
+status=$?
+if [ $status -eq 1 ]; then
```

Comment:
```suggestion
if [ $status -ne 0 ]; then
```

Reply:
I understood that we don't need a validation for the compiler version on the exploit code. So I will remove this condition from the assertion script, ok?